levels/management.tex

```diff
@@ -0,0 +1,235 @@
+\documentclass{article}
+
+\usepackage[table]{xcolor}
+\usepackage[paperwidth=525mm, paperheight=325mm]{geometry}
+
+% Don't show page numbers.
+\pagenumbering{gobble}
+
+% Column settings for skills able.
+\newcolumntype{C}{p{20em}}
+
+\begin{document}
+
+% Title of matrix.
+\begin{center}
+\begin{huge}
+Engineering Managment
+\end{huge}
+\end{center}
+
+\bigbreak
+
+% Alternating row colors.
+\rowcolors{2}{gray!80!gray!50}{gray!50!gray!20}
+
+% Padding for each cell.
+{\renewcommand{\arraystretch}{2}
+
+% Skills table.
+\begin{center}
+\begin{tabular}{|C|C|C|C|C|}
+\hline
+    \rowcolor{blue!60!black!10}
+    \textbf{Level}
+    &
+    \cellcolor{blue!25}
+    \textbf{Product Output}
+    &
+    \textbf{Communication/Writing}
+    &
+    \textbf{Team Development}
+    &
+    \textbf{People Development}
+    \\
+\hline
+    % Level
+    Team Lead
+    &
+
+    % Product Output
+    \cellcolor{blue!25}
+    Team ships software reasonably on time with low regressions. Consistently
+    assess and front-loads risk.
+
+    \bigbreak
+
+    Team has low firefighting load.
+
+    \bigbreak
+
+    Proposes quarterly goals based on customer feedback, platform performance,
+    and operational toil.
+    &
+
+    % Communication
+    Understands and can communicate team mission and quarterly focus to
+    engineers. Engineers all understand team and individual success criteria.
+
+    \bigbreak
+
+    Can deliver cross-team projects on time.
+
+    \bigbreak
+
+    Runs effective 1x1s with the team. Effective 1x1s should have a meeting log
+    attached to the meeting invite which is used to provide feedback with links to
+    the levels guide and development plans. In addition, each 1x1 should cover what
+    is being worked on, what is waiting review, and what is coming up next for the
+    engineer.
```

Comment:
What is covered in 1x1s and how the manager goes about them seem like implementation details. Can we find a way to frame the outcome we expect of effective 1x1s and be less prescriptive about the implementation?

There's tons of [maybe even contradictory] advice out there about effective 1x1s. I'd argue the best managers are the ones that recognize that there isn't a one-size-fits-all solution and tailor their approach to the individual. If we can focus more on why we want this (employee engagement, meeting our commitments, low turnover, etc) I think we can make room for more flexibility.

Reply:
I made the wording a bit more generic. Instead of being PR specific made it more about what is being worked on, what others are waiting on, and career development. What do you think?